data/modules/Scout/ScanDisplay.lua

```diff
@@ -183,35 +183,47 @@ function scanDisplay:drawScanInfo(scan, isHighlighted)
 	-- displayed when the active scanner cannot carry out the scan
 	local altitude = string.upper(ls.INVALID_SCANNER)
 	local target = ""
+	local completion = ""
 
 	local params = scanMgr:GetScanParameters(sBody, scan.minResolution, scan.orbital)
 	if params and params.canScan then
 		altitude = ui.Format.Distance(params.maxAltitude)
 	end
 
-	if scan.orbital then
-		target = string.format("%.1f%%", scan.targetCoverage * 100.0)
-	else
-		target = ui.Format.Distance(scan.targetCoverage * 1000.0, "%.1f")
-	end
-
-	local completion = math.min(1.0, scan.coverage / scan.targetCoverage)
+	target = ui.Format.Area(scan.targetCoverage * 1e6)
+	completion = ui.Format.Area(scan.coverage * 1e6)
```

Comment:
Sure; I left it this way to reduce the diff. `completion` can be removed entirely, but `target` is used multiple times.

And just a habit of mine to have all variables declared at the top of a scope - makes it (IMHO) easier to see where they all are and what a function might be using instead of when they are interspersed throughout.

Reply:
Fixed.  No commit ref as I'm squashing-and-rebasing to hopefully have this PR finalised :)

Reply:
I definitely understand the desire to have clearly declared variables - in the interest of consistency however, we tend not to have separately-declared variables unless a function result needs to be assigned to a mix of local and non-local variables, or the variable is set inside of a subordinate block scope.

This is partially because the practice is a lot more resistant to inadvertently using a variable before it's been initialized, and partially just a stylistic convention of "declare variables where needed".